lib/driver.rb

```diff
@@ -0,0 +1,76 @@
+require_relative 'csv_record'
+
+STATUSES = [:AVAILABLE, :UNAVAILABLE]
+
+module RideShare
+  class Driver < CsvRecord
+
+    attr_reader :name, :vin, :status, :trips
+
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: [])
+      super(id)
+      raise ArgumentError, 'Name must be a string' unless name.class == String
+      @name = name
+      raise ArgumentError, 'Incorrect VIN format' unless vin.length == 17 && vin.class == String
+      @vin = vin
+      raise ArgumentError, 'Incorrect status' unless STATUSES.include?(status)
```

Comment:
It's helpful to include what the invalid argument is in your messages:

```suggestion
      raise ArgumentError, 'Incorrect status: #{status}' unless STATUSES.include?(status)
```